code_challenges/advent_of_code/find_easter_bunny_hq.py

```diff
@@ -0,0 +1,56 @@
+def elves_directions():
+    with open(
+        "./code_challenges/advent_of_code/directions_to_easter_bunny_hq"
+    ) as directions:
+        moves = directions.read().split(",")
+    return moves
+
+
+def find_easter_bunny_hq():
+    moves = elves_directions()
+    north_south = 0
+    east_west = 0
+    direction = "N"
+
+    moves = [move.strip() for move in moves]
+    print(moves)
+    for move in moves:
+        left_or_right = move[0]
+        steps = move[1]   
```

Comment:
What does line 19 do when it hits a number move like `L193`?

Instead of:
```python
    moves = [move.strip() for move in moves]
    print(moves)
    for move in moves:
        left_or_right = move[0]
        steps = move[1]   
```
I'd try:
```python
    for move in moves:
        move = move.strip()
        left_or_right, steps = move[1], int(move[1:])
        if ....
```

Reply:
i knew i wasnt seeing something!